src/call.c

```diff
@@ -684,6 +687,18 @@ int call_alloc(struct call **callp, const struct config *cfg, struct list *lst,
 	if (msg && mbuf_get_left(msg->mb))
 		got_offer = true;
 
+	/* Check Call-Info header for ;answer-after=0 */
+	hdr = sip_msg_hdr(msg, SIP_HDR_CALL_INFO);
+	if (hdr) {
+		struct pl aprm;
+		if (0 == msg_param_decode(&hdr->val, "answer-after", &aprm)) {
```

Comment:
I suggest to use `pl_u32()` to convert the string to a number:

https://github.com/creytiv/re/blob/master/include/re_fmt.h#L31

and then use that value to start an async timer:

```c
uint32_t val;

val = pl_u32(&aprm);

tmr_start(&tmr, val, answer_after_tmr_handler, xxx);
```